add verbose to windows builds and use 4 parallel jobs (#1183)
Azure/azure-sdk-for-cpp: CONTRIBUTING.md

cmake --build .
```

> #### Static Analyze

Reviewer: ```suggestion
#### Static Analysis
```

> When the project is built using MSVC on Windows, the compiler can run [static analysis](https://docs.microsoft.com/cpp/code-quality/walkthrough-analyzing-c-cpp-code-for-defects) to the code. The CMake project can add the required compiler flags to perform this check. To enable this feature, set an environment variable `AZURE_ENABLE_STATIC_ANALYZE`.

Reviewer: ```suggestion
When the project is built using MSVC on Windows, the compiler can run [static analysis](https://docs.microsoft.com/cpp/code-quality/walkthrough-analyzing-c-cpp-code-for-defects) on the code. The CMake project can add the required compiler flags to perform this check. To enable this feature, set an environment variable `AZURE_ENABLE_STATIC_ANALYSIS`.
```

> Have in mind that enabling static analysis will impact building time severally.

Reviewer: ```suggestion
Keep in mind that enabling static analysis will impact building time significantly, so it is recommended to run it right before submitting the PR but not in your inner developer loop.
```

> Keep in mind that enabling static analysis will impact building time significantly, so it is recommended to run it right before submitting the PR but not in your inner developer loop.

Reviewer 2: ```suggestion
Keep in mind that enabling static analysis will significantly impact build time.  It is recommended to run it right before submitting the PR, but not in your inner developer loop.
```

> The static code analysis is `ON` for the CI pipelines. You can usually turn this feature `ON` locally to debug errors reported during CI or for the last time you build and test before creating a new PR.

Reviewer: ```suggestion
The static code analysis is `ON` for the CI pipelines. You can turn this feature `ON` locally to debug errors reported during CI or for the last time you build and test before creating a new PR.
```
